l10n_ch_payment_slip/static/src/js/qwebactionmanager.js

```diff
@@ -0,0 +1,37 @@
+odoo.define('l10n_ch_payment_slip.report', function (require) {
+    'use strict';
+
+    var ActionManager= require('web.ActionManager');
+    var crash_manager = require('web.crash_manager');
+    var framework = require('web.framework');
+
+
+    ActionManager.include({
+        ir_actions_report: function (action, options) {
+            var report_url = '';
+
+            if (action.report_type !== 'reportlab-pdf') {
```

Comment:
same for reportlab story

Reply:
@yvaucher  can you pls add some context about reportlab here?

Reply:
@i-vyshnevska reportlab was chosen to generate the pdf for precision issue with printers.

We have multiple parameters to align the payment slip and it's elements vertically and horizontally. Print-out might be altered by software configuration or printer setup if not printer hardware itself.

With Qweb it was way to hard to align correctly the payment slip and the Swiss post specs would make your payment slip rejected if you don't fix in a tiny error margin.

Nowadays, it may have improved and the payment slip as we know it will soon(-ish) be replaced (starting 30th of June 2020) by the QRCode version for which the readers will be more efficient.

Reply:
@yvaucher I  added it's to comments for clarifying, can we pass this PR further?